pkgs/apps/mrcc/default.nix

```
@@ -111,46 +62,13 @@ in stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
-  /*
-  The postFixup introduces a lot of wrapper magic:
-    1. All executables of MRCC are wrapped, so that correct MPI libraries and coreutils are found.
-    2. The original mrcc entry points (dmrcc and dmrcc_mpi) are moved (to dmrcc-original and
-       dmrcc_mpi-original), so that they can be replaced by other wrapper scripts
-       (dmrccEntryScript and dmrccMpiEntryScript).
-    3. The wrapper scripts dmrccEntryScript and dmrccMpiEntryScript are copied in the mrcc bin
-       directory and replace the original entry points. Those scripts create in the directory from
-       which they were called a "local" mrcc installation by linking the executables to calling
-       directory and replace all the absolute paths in the wrappers of this derivation.
-    4. The entrypoint wrapper scripts are wrapped AGAIN so that they have the correct coreutils
-       available at runtime.
-  */
   postFixup = ''
-    # Step 1:
-    # Look for all original mrcc executables and wrap them for paths and stuff
     exes=$(find $out/bin/ -type f -executable)
     for exe in $exes; do
     wrapProgram $exe \
       --prefix PATH : ${stdenv.cc.coreutils_bin}/bin \
```

```suggestion
      --prefix PATH : ${lib.getBin coreutils}/bin \
```
The coreutils from cc are a buildtime dependence.